tests/unit/eWiseMul.cpp

```diff
@@ -195,6 +221,9 @@ void grb_program( const size_t &n, RC &rc ) {
 		rc = SUCCESS;
 	}
 	if( rc != SUCCESS ) { return; }
+	if( nonblocking_execution ) {
+		rc = rc ? rc : set( left, 1 );
```

Comment:
The output error code of grb::set is never expected. Should it fail, the test should exit earlier or otherwise a misleading error will follow from the below subtest?